From 31f84751889857fed7e064c1ec713382582e435f Mon Sep 17 00:00:00 2001
From: Jakub Jirutka <jakub@jirutka.cz>
Date: Sun, 6 Feb 2022 16:10:34 +0100
Subject: [PATCH] Fix wrong _IOC_TYPE on musl

    error[E0308]: mismatched types
      --> libv4l-rs-917e1f4ffd90b94a/d7ac716/src/v4l2/api.rs:23:24
       |
    23 |         v4l2_ioctl(fd, request, argp)
       |                        ^^^^^^^ expected `u64`, found `i64`

diff --git a/src/v4l2/vidioc.rs b/src/v4l2/vidioc.rs
--- a/libv4l-rs/src/v4l2/vidioc.rs
+++ b/libv4l-rs/src/v4l2/vidioc.rs
@@ -1,11 +1,7 @@
 use crate::v4l_sys::*;
 
-#[cfg(not(target_env = "musl"))]
 #[allow(non_camel_case_types)]
 pub type _IOC_TYPE = std::os::raw::c_ulong;
-#[cfg(target_env = "musl")]
-#[allow(non_camel_case_types)]
-pub type _IOC_TYPE = std::os::raw::c_long;
 
 // linux ioctl.h
 const _IOC_NRBITS: u8 = 8;
